### ISSUE TITLE
[사전학습] 딥러닝 기초 개념 학습

### ISSUE THREAD

**Issue by YOONPYOGitHub** [YOONPYOGitHub/knowledge-distillation#13]
## 목표
딥러닝의 기본 개념을 팀원 전원이 이해하고 공유하기

## 학습 항목
- [ ] Neural Network 기본 구조 (입력층, 은닉층, 출력층)
- [ ] Activation Function (ReLU, Sigmoid, Softmax)
- [ ] Loss Function 개념 (Cross-Entropy, MSE)
- [ ] Optimizer 개념 (SGD, Adam, AdamW)
- [ ] Backpropagation 기본 원리
- [ ] Overfitting vs Underfitting
- [ ] Batch Size, Epoch, Learning Rate 의미

## 참고 자료
- `docs/glossary.md` — [딥러닝 기초 용어](#2-딥러닝-기초-용어) 섹션
- 3Blue1Brown - Neural Networks 시리즈 (YouTube)
- Andrew Ng - Deep Learning Specialization (Coursera)

## 완료 기준
- [ ] 각 팀원이 위 항목을 설명할 수 있을 때
- [ ] 용어집에서 모르는 용어가 없을 때

**Comment by YOONPYOGitHub** [comment]
새로운 1년 학습 로드맵 기반 이슈로 재설계합니다.
